src/sentry/static/sentry/app/views/releaseDetails.jsx

```diff
@@ -18,6 +21,53 @@ import TimeSince from 'app/components/timeSince';
 import Version from 'app/components/version';
 import withEnvironmentInQueryString from 'app/utils/withEnvironmentInQueryString';
 
+class ReleaseDetailsActions extends React.Component {
+  static propTypes = {
+    api: PropTypes.object.isRequired,
+    orgId: PropTypes.string.isRequired,
+    projectId: PropTypes.string.isRequired,
+    release: PropTypes.object.isRequired,
+  };
+
+  handleDelete = () => {
+    let {orgId, projectId, release, api} = this.props;
+    let path = `/organizations/${orgId}/releases/${release.version}/`;
```

Comment:
I believe `release.version` needs to be URI encoded.

Ideally, we have a better way of constructing these URLs without having to remember which parameter contains special characters or not.

Reply:
Yes, it does need urlencoding as it is encoded in other places. Have we ever considered adding `_links` properties to the API responses? That would allow the react code to use hypermedia controls instead of cooking up all the URLs clientside.

Reply:
Some responses do use `Link` properties, but I _think_ exclusively for pagination ([example](https://docs.sentry.io/api/events/get-project-group-index/)).